jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListBasicFiltersPresenter.java

```diff
@@ -16,20 +16,22 @@
 
 package org.jbpm.workbench.ht.client.editors.taskslist;
 
+import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
-
 import javax.inject.Inject;
 
 import org.dashbuilder.dataset.DataSetLookup;
 import org.dashbuilder.dataset.DataSetLookupFactory;
 import org.dashbuilder.dataset.sort.SortOrder;
 import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.jbpm.workbench.common.client.filters.active.ActiveFilterItem;
 import org.jbpm.workbench.common.client.filters.basic.BasicFiltersPresenter;
 import org.jbpm.workbench.ht.client.resources.i18n.Constants;
 
 import static org.dashbuilder.dataset.filter.FilterFactory.*;
-import static org.jbpm.workbench.ht.client.util.TaskUtils.*;
+import static org.jbpm.workbench.ht.client.util.TaskUtils.TaskType;
+import static org.jbpm.workbench.ht.client.util.TaskUtils.getStatusByType;
```

Comment:
@cristianonicolai  The compilation is launching this PMD failures to me: 
[INFO] PMD Failure: org.jbpm.workbench.ht.client.editors.taskslist.AbstractTaskListBasicFiltersPresenter:17 Rule:TooManyStaticImports Priority:3 Too many static imports may lead to messy code.
[INFO] PMD Failure: org.jbpm.workbench.ht.client.editors.taskslist.AbstractTaskListBasicFiltersPresenter:137 Rule:CollapsibleIfStatements Priority:3 These nested if statements could be combined.

Reply:
@nmirasch sorry, fixed now.